services/ui-auth/README.md

```diff
@@ -0,0 +1,28 @@
+# ui-auth
+
+## Configuration - AWS Systems Manager Parameter Store (SSM)
+
+The following values are used to configure the deployment of this service (see below for more background and context).
+| Parameter | Required? | Accepts a default? | Accepts a branch override? | Purpose |
+| --- | :---: | :---: | :---: | --- |
+| .../iam/path | N | Y | Y | Specifies the [IAM Path](https://docs.aws.amazon.com/IAM/latest/UserGuide/reference_identifiers.html#identifiers-friendly-names) at which all IAM objects should be created. The default value is "/". nb:The path variable in IAM is used for grouping related users and groups in a unique namespace, usually for organizational purposes.|
+| .../iam/permissionsBoundaryPolicy | N | Y | Y | Specifies the [IAM Permissions Boundary](https://docs.aws.amazon.com/IAM/latest/UserGuide/access_policies_boundaries.html) that should be attached to all IAM objects. nb: A permissions boundary is an advanced feature for using a managed policy to set the maximum permissions that an identity-based policy can grant to an IAM entity.|
+| sesSourceEmailAddress | N | Y | Y | The email address with which the apllication sends the email. This email address must be verified in SES.|
+| okta_metadata_url | N | Y | Y | The SAML Metadata url for Okta, gotten from Okta's admin console from the app.|
```

Comment:
okta_metadata_url does not accept a default value.  Only a branch specific value can be specified, just like cloudfront/route53 domain names.

Reply:
i have corrected this.